pkg/controller/secret/secret_controller.go

```diff
@@ -570,8 +592,8 @@ func (r *ReconcileSecret) parseConfigMaps(reqLogger logr.Logger, cmList *corev1.
 
 	// Default to alerting on all ^openshift-.* namespaces if either list is empty, potentially indicating a problem parsing configMaps
 	if len(managedNamespaces) == 0 ||
-	   len(ocpNamespaces)     == 0 ||
-	   len(addonsNamespaces)  == 0 {
+		len(ocpNamespaces) == 0 ||
+		len(addonsNamespaces) == 0 {
```

Comment:
this was done by my linter, not me directly

Reply:
Well at least the indent is now in sync with the two lines in 597 and 598 ^^